src/window_gui.h

```diff
@@ -327,7 +327,7 @@ struct Window : ZeroedMemoryAllocator {
 	NWidgetStacked *shade_select;    ///< Selection widget (#NWID_SELECTION) to use for shading the window. If \c NULL, window cannot shade.
 	Dimension unshaded_size;         ///< Last known unshaded size (only valid while shaded).
 
-	int scrolling_scrollbar;         ///< Widgetindex of just being dragged scrollbar. -1 if none is active.
+	int active_widget;               ///< Widgetindex of currently active widget (e.g. dragged scrollbar). -1 if none is active.
```

Comment:
The term used in other GUI frameworks (at least Win32) is mouse capture, when one widget requests to receive all mouse events, regardless of X/Y, until further notice.

Reply:
Where and how should I be using this term?

Reply:
"Active" widget sounds misleading to me, it sounds like it has to do with keyboard focus or highlight or similar. I'd suggest naming this `mouse_capture_widget` instead.